### ISSUE TITLE
iOS returns true even if the network was not found

### ISSUE THREAD

**Issue by vanlooverenkoen** [wonjerry/flutter_wifi_connector#14]
NEHotspotConfigurationManager.shared.apply(hotspotConfiguration) 

finished without an error. We should check if the user is connected to that specific ssid. in the completion handler.

**Comment by fbuireu** [comment]
In the meantime, while the PR is being merged, you can use @vanlooverenkoen's fork in your own projects my modifying the `pubspec.yml` like:

```
  wifi_connector:
    git:
      url: https://github.com/vanlooverenkoen/flutter_wifi_connector
      ref: combined
```

Be aware that now the `WifiConnector` class required a `securityType` `enum`.

**Comment by vanlooverenkoen** [comment]
Fixed by #15
